src/js/components/organisms/page-header/page-header.ts

```diff
@@ -60,13 +45,31 @@ export class PageHeader extends Vue implements PageHeaderProps {
         this.searchOpen = false;
     }
 
+    handleShowCompactMenu(show): void {
```

Comment:
```suggestion
    handleOpenCompactMenu(show): void {
```

Perhaps also name this open instead of show?